ludwig/contribs/mlflow/__init__.py

```diff
@@ -225,6 +225,14 @@ def _log_mlflow_loop(q: queue.Queue, log_artifacts: bool = True):
             # Break out of the loop if we're not going to log anything.
             break
 
+        if log_metrics is not None:
+            if log_metrics["llm_eval_examples"] is not None:
+                mlflow.llm.log_predictions(
+                    inputs=log_metrics["llm_eval_examples"]["inputs"],
+                    prompts=log_metrics["llm_eval_examples"]["targets"],
+                    outputs=log_metrics["llm_eval_examples"]["outputs"],
+                )
+                del log_metrics["llm_eval_examples"]
```

Comment:
Why is this necessary?

Reply:
Are you referring to `del log_metrics["llm_eval_examples"]`? This is so that mlflow doesn't try to log the llm_eval_examples in `mlflow.log_metrics`, which would cause an error.

Reply:
Makes sense. Could you add a comment about this in both places?